Python/Setup/swix/swix.generate.targets

```diff
@@ -49,10 +49,6 @@ package name=$([System.IO.Path]::GetFileNameWithoutExtension($(OutputName))) ver
 <WriteLinesToFile File="$(SwrFile).tmp" Lines="vs.dependencies
 $(_Dependencies)
 " Overwrite="no" Encoding="ASCII" Condition="$(_Dependencies) != ''" />
-
-    <ItemGroup>
-      <Package Include="$(SwrFile)" />
-    </ItemGroup>
   </Target>
 
   <Target Name="_GenerateSwrFromSources" Condition="@(SwrSource) != ''">
```

Comment:
That uses VSSDK deployment, not SWIX. We have a project dedicated to deploying templates for the experimental hive with its own vsixmanifest. You can't install the swix packages into the experimental hive.

---
In reply to: [69645463](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69645463) [](ancestors = 69645463)